### PR TITLE
Makes computers and machines use power.

### PR DESCRIPTION
**Pull request by moonheart08**
You shouldn't be able to clone a living person with 5W.
Also makes light bulbs use even more power.

:cl: moony
- tweak: Nanotrasen has run out of space magic that allows them to make computers and machines use only 5W. Station power usage has greatly increased, and engineers only have 7-8 minutes to get power up and running before things start to get a bit hairy.
- tweak: Lightbulbs are now 120W industrial bulbs, instead of 40W high efficiency ones.
- remove: AME no longer requires APC power to start, to make it a bit easier to pick up after engineers who refuse to engineer.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Power/Generation/ame.yml [hunk 1]
```diff
@@ -62,8 +62,8 @@
       input:
         !type:CableDeviceNode
         nodeGroupID: HVPower
-  - type: ApcPowerReceiver
-  - type: ExtensionCableReceiver
+  #- type: ApcPowerReceiver
+  #- type: ExtensionCableReceiver
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Structures/Power/Generation/ame.yml:66]: imo if you want to remove the apc power requirement, you should also remove it from the code of the amecontrollercomp

**Review by PaulRitter — CHANGES_REQUESTED** [review]
lgtm, BUT how do the stations powernets fare against this sudden increase in powerconsumption?

**Comment by moonheart08** [comment]
> lgtm, BUT how do the stations powernets fare against this sudden increase in powerconsumption?

Measured it in #contributors last night, Packed stays recoverable until 18 min into the round, and departments begin to lose power 8min in.
Saltern seemed to follow the same pattern, though i didn't sit around for 20min to see what dept depowers last, i just know it's fine for about 7-8 minutes.

File: Resources/Prototypes/Entities/Structures/Power/Generation/ame.yml (hunk 1 above)
Comment (moonheart08) [reply Resources/Prototypes/Entities/Structures/Power/Generation/ame.yml:66]: That just makes it more difficult to reintroduce the function later, and the AME Controller's code already accounts for the ApcPowerReceiver being missing.

Comment (moonheart08) [reply Resources/Prototypes/Entities/Structures/Power/Generation/ame.yml:66]: image

**Comment by PaulRitter** [comment]
> > lgtm, BUT how do the stations powernets fare against this sudden increase in powerconsumption?
> 
> Measured it in #contributors last night, Packed stays recoverable until 18 min into the round, and departments begin to lose power 8min in. Saltern seemed to follow the same pattern, though i didn't sit around for 20min to see what dept depowers last, i just know it's fine for about 7-8 minutes.

thats *way to fast* for being unrecoverable imo

**Comment by Peptide90** [comment]
Happy to pick this up at some point if you're busy / not available @moonheart08, up to you :)

**Pull request closed without merging**